remote-scripts/dpdk_compile_1.sh

```diff
@@ -0,0 +1,171 @@
+##
+## Revision:
+## v1.0.0 - ruqin - 7/5/2018 - Build the script
+##
+###############################################################################
+
+: '
+        <test>
+            <testName>dpdk_compile</testName>
+            <testID>ESX-DPDK-001</testID>
+            <setupScript>
+                <file>setupscripts\change_cpu.ps1</file>
+                <file>setupscripts\change_memory.ps1</file>
+                <file>setupscripts\add_vmxnet3.ps1</file>
+                <file>setupscripts\add_vmxnet3.ps1</file>
+                <file>setupscripts\add_vmxnet3.ps1</file>
+            </setupScript>
+            <testScript>dpdk_compile_1.sh</testScript>
+            <files>remote-scripts/dpdk_compile_1.sh</files>
+            <files>remote-scripts/utils.sh</files>
+            <RevertDefaultSnapshot>True</RevertDefaultSnapshot>
+            <testParams>
+                <param>VCPU=8</param>
+                <param>VMMemory=8GB</param>
+                <param>TC_COVERED=RHEL-136107</param>
+            </testParams>
+            <timeout>600</timeout>
+            <onError>Continue</onError>
+            <noReboot>True</noReboot>
+        </test>
+'
+
+
+dos2unix utils.sh
+
+#
+# Source utils.sh
+#
+. utils.sh || {
+    echo "Error: unable to source utils.sh!"
+    exit 1
+}
+
+#
+# Source constants file and initialize most common variables
+#
+UtilsInit
+
+###############################################################################
+##
+## Put your test script here
+## NOTES:
+## 1. Please use LogMsg to output log to terminal.
+## 2. Please use UpdateSummary to output log to summary.log file.
+## 3. Please use SetTestStateFailed, SetTestStateAborted, SetTestStateCompleted,
+##    and SetTestStateRunning to mark test status.
+##
+###############################################################################
+
+
+
+
+# Download compressed file from given url
+
+GetDistro
+
+LogMsg $DISTRO
+
+if [ "$DISTRO" == "redhat_7" ]; then
+    LogMsg "RHEL7"
+    UpdateSummary "RHEL7"
+    url=https://fast.dpdk.org/rel/dpdk-18.02.2.tar.xz
+elif [ "$DISTRO" == "redhat_8" ]; then
+    LogMsg "RHEL8"
+    UpdateSummary "RHEL8"
+    url=https://fast.dpdk.org/rel/dpdk-18.02.1.tar.xz
+else
+    SetTestStateSkipped
+    LogMsg "Not support rhel6"
+    exit 1
+fi
+
+LogMsg $url
+filename=$(curl -skIL $url | grep -o -E 'filename=.*$' | sed -e 's/filename=//')
+filename="$(echo -e "${filename}" | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//')"
+
+if [ ${#filename} -le 1 ]
+then
+    url_arr=(${url//\// })
+    filename="${url_arr[-1]}"
+fi
+
+LogMsg $filename
+
+if [ ! -f "$filename" ]
+then
+    LogMsg "Source Code Not Exist"
+    wget $url
+    if [ ! "$?" -eq 0 ]
+    then
+        LogMsg "Source Code Download Failed"
+        SetTestStateAborted
+        exit 1
+    fi
+    
+fi
+
+# Install required packages
+
+yum install elfutils-libelf-devel  elfutils-devel \
+make gcc glibc-devel kernel-devel numactl-devel numactl-libs python-devel -y
+
```

Comment:
1. Although line 118, line 124 needn't to check them right or not, as any error, will lead "make" process failed, but, if necessary, we can add these checks for them, as we can know the error easily